feat: add research transparency section (#145)
PovertyAction/ipa-research-data-science-hub: research-transparency/understanding-transparency.qmd

:::

## Defining Research Transparency

Research transparency refers to the practice of openly sharing the methods, data, and materials used in research studies so that others can understand, verify, and build upon scientific findings. At its core, transparency in research is about making the research process visible—from initial hypotheses through data collection and analysis to final results.

:::{.callout-note}

## IPA's Research Transparency Initiative
In 2014, IPA launched the Research Transparency Initiative to promote openness, transparency, and reproducibility of research. This Initiative positions IPA within a growing movement that includes funders, journals, and other research organizations, all working to encourage higher quality research practices. At IPA, we believe that high quality research goes hand in hand with research transparency.

Through the advocacy of registering studies and sharing materials such as data and code, other researchers, analysts, and academics can replicate and reanalyze results, strengthening the overall credibility of development economics research.

:::

> Research transparency is not simply about following rules or checking boxes—it's about creating an ecosystem where research findings can be trusted, verified, and used to inform policy decisions that affect people's lives. When we conduct randomized controlled trials to evaluate poverty alleviation programs, for example, the stakes are high. Policymakers and organizations rely on this evidence to make decisions about resource allocation. Transparent research practices ensure that these decisions rest on solid, verifiable foundations.

Reviewer: ```suggestion
Research transparency is not simply about following rules or checking boxes; it is about creating an ecosystem where research findings can be trusted, verified, and used to inform policy decisions that affect people's lives. When we conduct randomized controlled trials to evaluate poverty alleviation programs, for example, the stakes are high. Policymakers and organizations rely on this evidence to make decisions about resource allocation. Transparent research practices ensure that these decisions rest on solid, verifiable foundations.
```

Research transparency practices directly address each of the problems described above through concrete mechanisms and tools.

### Trial Registration and Pre-Analysis Plans

**Trial registration** involves publicly recording key information about a study before it begins. Registering a trial in the [American Economic Association's](https://www.socialscienceregistry.org/) Registry for Randomized Controlled Trials (AEA RCT Registry) or [ClinicalTrials.gov](https://clinicaltrials.gov/) creates a public record of the study's existence, combating publication bias.

When studies are registered in advance:

- The research community knows the study is happening, even if results are never published
- It becomes harder to bury null results in the "file drawer"
- Systematic reviews can search registries to find unpublished studies

**Pre-analysis plans** (PAPs) are more detailed documents that specify the study's hypotheses, primary and secondary outcomes, and planned analyses before data analysis begins. Pre-analysis plans combat p-hacking by creating a public record of what the researchers intended to test and how they intended to test it.

When researchers commit to a pre-analysis plan:

> - It's clear which outcomes were planned and which were explored post-hoc

Reviewer: ```suggestion
- It is clear which outcomes were planned and which were explored post-hoc
```

- **Secondary use**: Data can be used to answer new questions
- **Learning**: Students and early-career researchers can learn from real data
- **Efficiency**: Resources are not wasted recreating data that already exist

Data sharing promotes the development of better quality materials by project teams. When researchers know their data will be public, they have additional incentive to ensure quality, clear documentation, and proper organization.

:::{.callout-note}

## IPA's Data Repository

IPA has created its own data repository using Harvard's Dataverse platform. The repository makes data from IPA studies publicly accessible while ensuring proper documentation and ethical protection of participants. Visit [IPA's Dataverse](https://dataverse.harvard.edu/dataverse/IPA) to explore available datasets.

:::

Data sharing must balance transparency with ethical obligations to protect research participants. Personally identifiable information (PII) must be removed before data is shared publicly. Despite this necessary limitation, anonymized data can still serve all the functions described above.

## Transparency's Role in Development Policy

> Research transparency is not just a set of individual practices—it's an ecosystem involving multiple stakeholders working together to establish and maintain higher standards.

Reviewer: ```suggestion
Research transparency is not just a set of individual practices; it's an ecosystem involving multiple stakeholders working together to establish and maintain higher standards.
```
